### PR TITLE
DR-118 readthedocs fixes

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://semi-technology.atlassian.net/browse/DR-118)

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -99,7 +99,7 @@ Improvements include:
 Fixes include:
 - The potential for deadlocks and data races when batching has been reduced.
 - Fixes a number of missing properties and poor docstrings in ``weaviate.connect_to_x`` methods.
-- Adds the missing ``offset`` paramater to all queries.
+- Adds the missing ``offset`` parameter to all queries.
```
Comment (daveatweaviate) [inline docs/changelog.rst:102]: Some unrelated typos in this file

File: weaviate/collections/queries/base.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def __deserialize_non_ref_prop(self, value: properties_pb2.Value) -> Any:
         if value.HasField("null_value"):
             return None
 
-        _Warnings.unkown_type_encountered(value.WhichOneof("Value"))
+        _Warnings.unknown_type_encountered(value.WhichOneof("Value"))
```
Comment (daveatweaviate) [inline weaviate/collections/queries/base.py:194]: typo in the method name: unkown -> unknown

File: weaviate/warnings.py [hunk 3]
```diff
@@ -231,9 +231,9 @@ def batch_rate_limit_reached(msg: str, seconds: int) -> None:
         )
 
     @staticmethod
-    def unkown_type_encountered(field: str) -> None:
+    def unknown_type_encountered(field: str) -> None:
```
Comment (daveatweaviate) [inline weaviate/warnings.py:234]: As above

**Review by daveatweaviate — COMMENTED** [review]
@dirkkul @tsmith023  This should fix the readthdocs page.

I'm not sure if it needs to be backported to earlier releases. If so, I'll do that tomorrow. Similarly, I don't see the versioin picker at the lower right of the ToC panel in my staging build. I think that might be an artifact though. 

Once we push this, I'll check if on backporting and the version picker

**Review by tsmith023 — APPROVED** [review]

**Comment by tsmith023** [comment]
>I'm not sure if it needs to be backported to earlier releases

AFAIK, this is impossible since the builds are executed on the released GH tags that must be static and unchanging

**Pull request merged**